src/analyze/analyze.c

```diff
@@ -91,6 +91,7 @@ DotMode arg_dot = DEP_ALL;
 char **arg_dot_from_patterns = NULL, **arg_dot_to_patterns = NULL;
 usec_t arg_fuzz = 0;
 PagerFlags arg_pager_flags = 0;
+bool arg_tldr = false;
```

Comment:
why not just make this CatFlags? converting between the bool and the flags seems unnecessary to me. we don't need that layer of abstraction... it would then be nicely symmetric t PageFlags and suchlike.

(but doesn't really matter)

Reply:
Yeah, you're right. I reworked it to just pass the flags. (The lower layer has knownledge whether sections are used in a specific file format, so the flags parameter is extended with that later on.)